src/components/common/CenteredAlert.tsx

```diff
@@ -0,0 +1,40 @@
+import { Grid, Alert, AlertProps } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import React from 'react';
+
+const useStyles = makeStyles(() => ({
+  grid: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyItems: 'center',
+    justifyContent: 'center',
+    minHeight: '100vh',
+  },
+  alert: {
+    padding: '20px',
+    display: 'flex',
+    alignItems: 'center',
+    fontSize: '16px',
+    borderRadius: '10px',
+    '& div': {
+      padding: 0,
+    },
+    '& .MuiAlert-action': {
+      paddingLeft: '40px',
+      paddingRight: '10px',
+    },
+  },
+}));
+
+/** Message occupying full screen and displaying slightly customized MUI Alert component */
+function CenteredAlert(props: AlertProps) {
+  const classes = useStyles();
+
+  return (
+    <Grid className={classes.grid}>
```

Comment:
Maybe it is better to use the `Box` component here with its properties instead of using Grid with custom styles. If you have some time try to do it that way.

Reply:
I tried that initially but because of more advanced CSS rules, this looked cleaner.